File: Java/Beginner/1036/Main.java
import java.util.Scanner;
import java.lang.Math;

class Main {

    public static void main(String args[]) {
        Scanner scan = new Scanner(System.in);

        //coeficients
        double a = scan.nextDouble(), b = scan.nextDouble(), c = scan.nextDouble();
        //roots and discriminant
        double x1, x2, discriminant;
        discriminant = (b*b) - (4*a*c);
            if((discriminant < 0) || (a == 0.0)) System.out.println("Impossivel calcular");     //no real roots 
            else {
                double delta = Math.sqrt(discriminant);
                x1 = (-1*b + delta)/(2*a);
                x2 = (-1*b - delta)/(2*a);        
                System.out.printf("R1 = %.5f\n", x1);
                System.out.printf("R2 = %.5f\n", x2);

            }

        scan.close();
    }
}

File: C/Beginner/1036/main.c
#include <stdio.h>
#include <math.h>

int main(void) {

    float a, b, c, r1, r2, discriminant;
    scanf("%f%f%f", &a,&b, &c);

    discriminant = sqrt(pow(b,2) - 4*a*c);

    if(a != 0 && discriminant >= 0) {       //a = 0 -> não tem expoente 2 || não existe raiz quadrada real de numero negativo

        r1 = (-b + discriminant) / (2*a);
        r2 = (-b - discriminant) / (2*a);

        printf("R1 = %.5f\nR2 = %.5f\n", r1, r2);
    }
    else printf("Impossivel calcular\n");

    return 0;
}

//  [-b +- sqrt(b^2 - 4ac)] / 2a
